src/extension.ts

```diff
@@ -33,9 +33,9 @@ function runCalculate ( editMaker : IEditMaker ){
 		}
 
 		activeTextEditor.edit( (textEditorEdit) => {
-			activeTextEditor.selections.forEach( (selection) => {
+			activeTextEditor.selections.forEach((selection, index) => {
 
-				var selectedText = activeTextEditor.document.getText(selection);
+				var selectedText = activeTextEditor.document.getText(selection).replace(/\$i/g, index + 1);

```

Comment:
Getting an error on build on both the master you PR'd against and the updated master:
`src/extension.ts(38,85): error TS2345: Argument of type 'number' is not assignable to parameter of type '(substring: string, ...args: any[]) => string'.`

Reply:
I've added explicit String cast and hope now is ok.